router/batchrouter/batchrouter.go

```diff
@@ -333,13 +334,28 @@ func (brt *HandleT) copyJobsToStorage(provider string, batchJobs BatchJobsT, mak
 	}
 
 	brt.logger.Debugf("BRT: Starting upload to %s", provider)
-
-	var keyPrefixes []string
+	folderName := ""
 	if isWarehouse {
-		keyPrefixes = []string{config.GetEnv("WAREHOUSE_STAGING_BUCKET_FOLDER_NAME", "rudder-warehouse-staging-logs"), batchJobs.BatchDestination.Source.ID, time.Now().Format("01-02-2006")}
+		folderName = config.GetEnv("WAREHOUSE_STAGING_BUCKET_FOLDER_NAME", "rudder-warehouse-staging-logs")
+	} else {
+		folderName = config.GetEnv("DESTINATION_BUCKET_FOLDER_NAME", "rudder-logs")
+	}
+	dateFormat, err := uploader.GetStorageDateFormat([]string{folderName, batchJobs.BatchDestination.Source.ID}...)
+	datePrefixLayout := ""
```

Comment:
init with `datePrefixLayout := dateFormat` and just have the `if` condition below ?